### ISSUE TITLE
SCA: Define maskSpecificationFileForSCA within GUI

### ISSUE THREAD

**Issue by jpellman** [FCP-INDI/C-PAC#453]
The CPAC GUI presently does not have an obvious way to define the path to the SCA mask specification file.  We might want to add another field for this under "SCA Options".

**Comment by jpellman** [comment]
Actually, this is within ROI Average TSE Options, although it's worth considering if there might be a way of making it more clear in "SCA Options" that setting ROI Average TSE Options is a dependency.

**Comment by sgiavasis** [comment]
Definitely.. there are multiple areas in the pipeline configuration editor that would benefit from a clearer progression of requirements between settings. This is something we'd like to address in the near future in terms of either a different layout or having real-time pipeline configuration validation.

As for now, the "Test Configuration" feature in the pipeline configuration editor can help users catch missed inputs before they kick off any pipeline runs.
